native_client/deepspeech.h

```diff
@@ -163,6 +163,10 @@ int DS_EnableExternalScorer(ModelState* aCtx,
 /**
  * @brief Add a hot-word and its boost.
  *
+ * Positive value boosting increases and negative reduces chance of a word occuring in a transcription.
```

Comment:
This should be after the part `@param boost The boost.`

Reply:
```suggestion
 /**
 * @brief Add a hot-word and its boost.
 *
 * @param aCtx The ModelState pointer for the model being changed.
 * @param word The hot-word. Words that don't occur in the scorer (e.g. proper nouns) or strings that contain spaces won't be taken into account.
 * @param boost The boost. Positive value boosting increases and negative reduces chance of a word occuring in a transcription. Excessive positive boost might lead to splitting up of letters of the word following the hot-word.
 *
 * @return Zero on success, non-zero on failure (invalid arguments).
 */
```

Put everything at @param info like this? Or separately like this:

```suggestion
/**
 * @brief Add a hot-word and its boost.
 *
 * @param aCtx The ModelState pointer for the model being changed.
 * @param word The hot-word.
 * @param boost The boost. 
 *
 * Positive value boosting increases and negative reduces chance of a word occuring in a transcription.
 * Excessive positive boost might lead to splitting up of letters of the word following the hot-word.
 * Words that don't occur in the scorer (e.g. proper nouns) or strings that contain spaces won't be taken into account.
 *
 * @return Zero on success, non-zero on failure (invalid arguments).
 */
```